src/main/java/org/thoughtcrime/securesms/ShareActivity.java

```diff
@@ -206,13 +207,9 @@ public void onMediaResolved(Uri uri) {
   }
 
   private void handleResolvedMedia(Intent intent) {
+    int       accId            = intent.getIntExtra(EXTRA_ACC_ID, -1);
     int       chatId           = intent.getIntExtra(EXTRA_CHAT_ID, -1);
 
-    String shortcutId = intent.getStringExtra(ShortcutManagerCompat.EXTRA_SHORTCUT_ID);
-    if (chatId == -1 && shortcutId != null) {
-      chatId = Integer.parseInt(shortcutId);
-    }
-
```

Comment:
wondering where this came from, looks useless :)

Reply:
Yes, couldn't find any similar usage in the Internet, maybe we were using the intent ID to pass the chat id at the beginning so this was kept as a migration step or it was from android <=4 times

Reply:
And we don't even do it for other shortcuts we create